benchmarks/query_benchmarks/query_update/benchmark.py

```diff
@@ -0,0 +1,13 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryUpdate:
+    def setup(self):
+        bench_setup(migrate=True)
+
+    def teardown(self):
+        Book.objects.all().delete()
+
+    def time_query_update(self):
+        Book.objects.all().update(title="z")
```

Comment:
Can we repeat this 10x as well? Maybe it needs a different title each time? 

https://github.com/smithdc1/django-asv/runs/6720503551?check_suite_focus=true#step:5:68